build.sh

```diff
@@ -479,7 +480,7 @@ git_fetch_factorcode() {
 }
 
 cd_factor() {
-    cd factor
+    cd "factor"
```

Comment:
Here's the other quote I don't get. It's not wrong as such, but I don't get why we need them here.

Reply:
It's like the airquotes we always do with our fingers when we talk about "Factor".